code/auxiliaries/DefaultPopulationSettings.m

```diff
@@ -6,7 +6,7 @@
 %       is used in function PKSimCreatePopulation   
 
 % Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org 
-% Date: 12-jan-2017
+% Date: 30-apr-2018
```

Comment:
Why do we have dates here? This can only go wrong

Reply:
My suggestion would be to remove dates in those files. This does not make much sense to add a date in a source code file